packages/server/lib/browsers/firefox_automation.md

```diff
@@ -1,35 +1,39 @@
-# GeckoDriver
+# Firefox Automation
 
-## Purpose
+## GeckoDriver / WebDriver Classic
 
-Cypress uses [GeckoDriver](https://firefox-source-docs.mozilla.org/testing/geckodriver/index.html) to drive [classic WebDriver](https://w3c.github.io/webdriver.) methods, as well as interface with Firefox's [Marionette Protocol](https://firefox-source-docs.mozilla.org/testing/marionette/Intro.html). This is necessary to automate the Firefox browser in the following cases:
+### Purpose
 
-* Navigating to the current/next spec URL via [WebDriver Classic](https://w3c.github.io/webdriver.).
+Cypress uses [GeckoDriver](https://firefox-source-docs.mozilla.org/testing/geckodriver/index.html) to drive [classic WebDriver](https://w3c.github.io/webdriver) methods, as well as interface with Firefox's [Marionette Protocol](https://firefox-source-docs.mozilla.org/testing/marionette/Intro.html). This is necessary to automate the Firefox browser in the following cases:
```

Comment:
Can I ask for which reason you still need the Marionette protocol? I assumed that Marionette is no longer used since the move from marionette-js to geckodriver.

Reply:
@whimboo we are using `marionette` through `geckodriver` since we need to install our web extension  unsigned to automate a few additional things in the browser that we do not get with CDP. Based on the [source code](https://searchfox.org/mozilla-central/rev/cc01f11adfacca9cd44a75fd140d2fdd8f9a48d4/testing/geckodriver/src/command.rs#33-40) and the browser debug logs, this looks to use marionette under the hood. We are just abstracting the usage through webdriver.

Reply:
Well, that is just our WebDriver classic implementation with the vendor extension to install WebExtensions. As such you shouldn't have to call this out separately given that geckodriver acts as a proxy for that command.

Reply:
I can add an edit into the changelog that just states we are just using webdriver. Might be simpler at least for user facing details